.eslintrc.js

```diff
@@ -28,6 +28,8 @@ module.exports = {
                 args: 'after-used',
             },
         ],
+        "react/display-name": "off",
+
```

Comment:
Looks good 👍 might be worth dropping comments that disable this rule where it was used?

Reply:
That's now done ✅